apps/web/src/app/layout.tsx

```diff
@@ -14,21 +14,25 @@ const calSans = LocalFont({
   variable: "--font-calsans",
 });
 
+const TITLE = "Open-source monitoring service";
```

Comment:
For Title I would use the product name 
it would be better to have `Blog - OpenStatus`